docs/examples/hello-world/manifests.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
 spec:
   ingressClassName: ngrok
   rules:
-    - host: ${NGROK_SUBDOMAIN}.ngrok.io
+    - host: ${NGROK_SUBDOMAIN}.ngrok.app
```

Comment:
Maybe don't update this, or update the md file as well so they remain consistent https://github.com/ngrok/kubernetes-ingress-controller/blob/main/docs/examples/hello-world/README.md